framework/dotnet/BenchPress.Module/BenchPress.Module.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This folder will be for Powershell so we dont need the csproj or program.cs file